public/app/percona/shared/helpers/notification-manager.ts

```diff
@@ -0,0 +1,17 @@
+import { appEvents } from 'app/core/app_events';
```

Comment:
This file can be removed. Use appEvents directly. Check Kubernetes.hooks.ts as reference.

Reply:
done